### PR TITLE
mailmap,orgnization: Update sangfor affiliation

### PR DESCRIPTION
**Pull request by Aran85**
Signed-off-by: Zengran Zhang <zhangzengran@sangfor.com.cn>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by Aran85** [comment]
@tchaikov would you please have a review? Thanks! :D

**Comment by joscollin** [comment]
retest this please

**Comment by tchaikov** [comment]
retest this please

**Comment by tchaikov** [comment]
retest this please

File: .mailmap [hunk 1]
```diff
@@ -500,9 +508,12 @@ Yuri Weinstein <yuriw@redhat.com> <yuriw@magna002.ceph.redhat.com>
 Yuri Weinstein <yuriw@redhat.com> <yweins@redhat.com>
 Yuri Weinstein <yuriw@redhat.com> <yuri.weinstein@redhat.com>
 Xianxia Xiao <xianxia.xiao@kylin-cloud.com>
+XiaoCheng Wang <wangxiaocheng@sangfor.com.cn>
+Xiaofei Cui <cuixiaofei@sangfor.com.cn> <cuixiaofei@sangfor.com>
 Zack Cerza <zack@redhat.com> <zack.cerza@inktank.com>
 Zack Cerza <zack@redhat.com> <zack@cerza.org>
-Zengran Zhang <zhangzengran@h3c.com>
```
Comment (tchaikov) [inline .mailmap]: i think we should keep as it is. as h3c sponsored the commits contributed by `zhangzengran@h3c.com` when `zhangzengran@h3c.com` was affiliated to H3C.

and the new  and newly added `zhangzengran@sangfor.com.cn` entry will help point this mail address to @zhangsw .

File: .organizationmap [hunk 2]
```diff
@@ -205,7 +205,6 @@ H3C <contact@h3c.com> Yehua Chen <chen.yehua@h3c.com>
 H3C <contact@h3c.com> Yongqiang He <he.yongqiang@h3c.com>
 H3C <contact@h3c.com> Yue Zhu <zhu.yue@h3c.com>
 H3C <contact@h3c.com> Yunhui Chen <chen.yunhui@h3c.com>
-H3C <contact@h3c.com> Zengran Zhang <zhangzengran@h3c.com>
```
Comment (tchaikov) [inline .organizationmap]: might want to keep this link.

**Comment by Aran85** [comment]
@tchaikov  thanks, i got it, updated.

**Pull request merged**